base/src/main/java/org/aya/tyck/LittleTyper.java

```diff
@@ -77,6 +78,7 @@ public record LittleTyper(@NotNull TyckState state, @NotNull LocalCtx localCtx)
       case FormTerm.Univ univ -> new FormTerm.Univ(univ.lift() + 1);
       case FormTerm.Interval interval -> new FormTerm.Univ(0);
       case PrimTerm.End end -> FormTerm.Interval.INSTANCE;
+      case PrimTerm.Str str -> state.primFactory().getOption(PrimDef.ID.STR).get().result();
```

Comment:
Are you sure it's `.get().result()` not `getCall(ID.STR)`?

Reply:
No it's just I forgot.